plugins/catalog-unprocessed-entities/src/components/PendingEntities.tsx

```diff
@@ -84,6 +96,9 @@ export const PendingEntities = () => {
         options={{ pageSize: 20 }}
         columns={columns}
         data={data?.entities || []}
+        onSearchChange={(searchTerm: string) =>
+          setSelectedSearchTerm(searchTerm)
```

Comment:
is this here only to trigger a repaint of this component? is it really needed? we don't read the state after all

Reply:
yes, copied from how other Table components deal with search/filter